README.md

```diff
@@ -38,14 +39,28 @@ Add somewhere along:
   end
 ```
 
-In the tests import the `TestSelector.Test.Helpers`:
+**Testing with Floki**
+In the tests import the `TestSelector.Test.HoundHelpers` :
```

Comment:
Or in a test case (i.e. DataCase or ConnCase) to skip having to import it in every test